sdk/utiltities_test.go

```diff
@@ -0,0 +1,72 @@
+package sdk
```

Comment:
Wrong name of the file, `utiltities` --> `utilities`

Reply:
yeah, renamed it